src/Sylius/Component/Core/Test/Services/SecurityServiceInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Test\Services;
+
+use Behat\Mink\Session;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+interface SecurityServiceInterface
```

Comment:
We should figure out a better name later for this service, but I am fine with this in first iteration.